qa/tasks/qemu.py

```diff
@@ -635,7 +635,7 @@ def task(ctx, config):
                         action: create / clone / none (optional, defaults to create)
                         image_name: <image name> (optional)
                         parent_name: <parent_name> (if action == clone),
-                        type: filesystem / block (optional, defaults to fileystem)
+                        type: filesystem / block (optional, defaults to filesystem)
```

Comment:
There are those who insist on "file system" but I still think "filesystem" is preferable.